products/checkbook/build_scripts/export.py

```diff
@@ -1,39 +1,17 @@
 from datetime import date
-from pathlib import Path
+
 from dcpy.connectors.edm.publishing import upload
 from dcpy.utils.git import git_branch
 
-from . import (
-    OUTPUT_DIR,
-    SOURCE_DATA_VERSIONS_FILENAME,
-    BUILD_OUTPUT_FILENAME,
-    SUMMARY_STATS_DESCRIBE_FILENAME,
-    SUMMARY_STATS_LOG_FILENAME,
-)
+from . import OUTPUT_DIR
 
 PUBLISHING_FOLDER = "db-checkbook"
 
 
 def run_export() -> None:
     build_environment = git_branch()
     version = str(date.today())
-    file_paths = [
-        Path(OUTPUT_DIR.name) / SOURCE_DATA_VERSIONS_FILENAME,
-        Path(OUTPUT_DIR.name) / BUILD_OUTPUT_FILENAME,
-        Path(OUTPUT_DIR.name) / SUMMARY_STATS_DESCRIBE_FILENAME,
-        Path(OUTPUT_DIR.name) / SUMMARY_STATS_LOG_FILENAME,
```

Comment:
they're the two least import files, but is it expected that `SUMMARY_STATS_DESCRIBE_FILENAME` and `SUMMARY_STATS_LOG_FILENAME` aren't in the S3 exports anymore?

Reply:
I ran manually - I just didn't generate this time (didn't run summarization fiel). I'll regenerate via gha when this is done, just can't publish since it has no workflow_dispatch since it hasn't been merged yet